code/network/multi_obj.h

```diff
@@ -32,19 +35,80 @@ struct net_player;
 #define OOC_AFTERBURNER_ON			(1<<5)
 // two spots now left for more OOC flags
 
+// what we are setting the subsystem "index" to if there was no subsystem.
+#define OOC_INDEX_NULLPTR_SUBSYSEM 255
+
+// Cyborg17, Server will be tracking only the last second of frames
+#define MAX_FRAMES_RECORDED		30
+#define PRIMARY_PACKET_CUTOFF			2000
+
 // and two special values to help with multilock
 #define OOC_INDEX_NULLPTR_SUBSYSEM 255
```

Comment:
Unless my eyes are messed up this is just the same #define as above. Surprised a compiler warning didn't get triggered here

Reply:
One came in from the multilock changes, and one accidentally from this patch since it should be already defined.

Reply:
I figured that would be the case. And a reminder to update the multilock packets to use short/ushort too.